api/api-javascript/api-git/src/main/resources/META-INF/dirigible/git/extensions/git.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module "@dirigible/git" {
+    module client {
+
+
+    }
+}
```

Comment:
I guess the `d.ts` file content would be added at later point?

Reply:
yes